filebeat/module/traefik/access/ingest/pipeline.json

```diff
@@ -1,71 +1,73 @@
 {
-    "description": "Pipeline for parsing Traefik access logs. Requires the geoip and user_agent plugins.",
```

Comment:
Changes from spaces to tabs? Not sure if we have a guideline for JSON files, but I'd keep it with spaces as it  seems to be the current convention in filebeat ingest pipelines.

Reply:
Done :)